deploy/images/controller/Dockerfile

```diff
@@ -1,17 +1,20 @@
-# Use distroless image which already includes ca-certificates
-FROM gcr.io/distroless/static:nonroot
```

Comment:
Why are we changing the base image?

Reply:
bicep CLI doesn't run on distroless or alpine. I think it's out of scope of this work to trial and error until I can get it to work. my solution for now is to temporarily use `ubuntu` image and file an issue to investigate and replace it when we can.